src/sage/groups/braid.py

```diff
@@ -446,11 +451,20 @@ def permutation(self, W=None):
             sage: B.<s0,s1,s2> = BraidGroup()
             sage: S = SymmetricGroup(4)
             sage: b = s0*s1/s2/s1
-            sage: b.permutation(W=S)
+            sage: c0 = b.permutation(W=S); c0
             (1,4,2)
-            sage: c = b.permutation(W=Permutations(4)); c
+            sage: c1 = b.permutation(W=Permutations(4)); c1
             [4, 1, 3, 2]
-            sage: c == b.permutation()
+            sage: c1 == b.permutation()
+            True
+
+        The canonical section from the symmetric group to the braid group
+        (sending a permutation to its associated permutation braid)
+        can be recovered.::
```

Comment:
```suggestion
        The canonical section from the symmetric group to the braid group
        (sending a permutation to its associated permutation braid)
        can be recovered::
```
I don't know that area of the literature as well as you, but I hadn't seen that terminology before. This is also better to the novice with "associated" added. Thank you. Anyways, just need to remove the period/full-stop as the outputted doc is "can be recovered.:"